deploy/docker/server.py

```
@@ -471,6 +481,59 @@ async def crawl_stream(
+                "Status": "Script executed with output"
+            }
+            raise HTTPException(400, response_data) 
+
+        return JSONResponse(output)
+    except subprocess.CalledProcessError as e:
+        # Return the command execution error details
+        response_data = {
+            "Status": "Failed to execute the script",
+            "Details": str(e)
+        }
+        raise HTTPException(400, response_data)     
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)